theano/gpuarray/blas.py

```diff
@@ -1682,7 +1787,7 @@ def make_node(self, kern, topgrad, shape=None):
             broadcastable = [topgrad.type.broadcastable[0], False,
                              False, False, False]
         else:
-            broadcastable = [topgrad.type.broadcastable[0], kern.type.broadcastable[1],
```

Comment:
(The failing test is `theano.gpuarray.tests.test_abstractconv.TestCorrMMConv3d.test_all`.)